src/app/components/flow-chart-editor/flow-chart-editor.component.ts

```diff
@@ -30,6 +32,10 @@ export class FlowChartEditorComponent {
   selectionInModel: NodeOrEdgeSelection = new NodeOrEdgeSelection
   showNodeTextInDrawing: boolean = true
   choiceShowNodeTextInDrawing: NodeCaptionChoice = this.updateShowNodeTextInDrawing()
+  layerNumberAlgorithms: {key: LayerNumberAlgorithm, value: string}[] = [
+    {key: LayerNumberAlgorithm.FIRST_OCCURING_PATH, value: 'first occuring path'},
```

Comment:
Misschien kan je dit const maken en buiten de class plaatsen. Dan zie je makkelijker dat dit geen member variabele is die van waarde kan veranderen.

Reply:
ik heb het field nu readonly gemaakt, ik weet niet hoe ik het nog beter kan maken omdat dit field in de html wordt gebruikt, en daarvoor moet het altijd een field van een instance zijn